compiler-core/src/language_server/engine.rs

```diff
@@ -427,6 +429,35 @@ where
 
         completions
     }
+
+    fn completion_imports<'b>(&'b self, module: &'b Module) -> Vec<lsp::CompletionItem> {
+        let mut completions = vec![];
+
+        // Module functions
+        for (module_name, dep_module) in &self.compiler.modules {
+            if module_name == &module.name {
+                continue;
+            };
+            let doc_string: String = dep_module.get_module_comments().join("\n");
```

Comment:
These will commonly be thousands of words, it's not suitable for displaying in the LSP. Let's not include this.